CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/AnalyticsScreen.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  AnalyticsScreen.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+
+enum AnalyticsScreen: String {
+    case none
```

Comment:
[Clarification]: when do we plan to use `none`? 🧐

Reply:
Theoretically no. I was thinking to have a value as a fallback if we need it.

Reply:
I will remove it.